utm.yaml

```diff
@@ -969,10 +986,22 @@ components:
 
             Start and end times, as well as lower and upper altitudes, are required for each
             volume. The end time may not be in the past.
-          minItems: 1
+
+            Required with at least one item when the operational intent is Accepted, Activated, or Nonconforming.
```

Comment:
This may be me just coming up to speed with the details of the off_nominal_volumes concept, but I thought that when an operation transitioned to the Contingent state, the USS was required to remove the volumes content from the OperationalIntentDetails. 

If this is the case, should we include another comment here that indicates that?

Reply:
Sounds good; added.